packages/conform-dom/form.ts

```diff
@@ -623,7 +623,7 @@ function shouldNotify<Schema>(
 	return false;
 }
 
-export function createFormContext<
```

Comment:
What does this helps comparing to setting an alias on the entry file?

Reply:
you as the author know its unstable, but as the contributor I might not if I don't look at the export boundary (index) that I would read this as stable code